src/vm/dllimport.cpp

```diff
@@ -6132,23 +6134,152 @@ bool         NDirect::s_fSecureLoadLibrarySupported = false;
 #endif // !FEATURE_PAL
 
 // static
-NATIVE_LIBRARY_HANDLE NDirect::LoadLibraryFromPath(LPCWSTR libraryPath)
+NATIVE_LIBRARY_HANDLE NDirect::LoadLibraryFromPath(LPCWSTR libraryPath, BOOL throwOnError)
 {
     STANDARD_VM_CONTRACT;
 
+    if (libraryPath == NULL)
```

Comment:
The usual way to do argument validation to do it as first thing in the publicly exposed method (e.g. either the C# method or QCall). Not bury it in the callers.

Reply:
@jkotas  I've made these changes in https://github.com/dotnet/coreclr/pull/21148